modules/output/results.py

```diff
@@ -124,6 +124,7 @@ def create_trade_info_table(self: MainResults, currency_symbol, justification) -
                              colorize(round(
                                  self.worst_trade_currency_amount, 2),
                                  0, str(currency_symbol)))
+    trade_info_table.add_row('Risk/ reward ratio', str(round(self.risk_reward_ratio, 2)))
```

Comment:
```suggestion
    trade_info_table.add_row('Risk / reward ratio', str(round(self.risk_reward_ratio, 2)))
```

Or 'risk vs reward ratio'? What do you think?